charts/openshift-etcd-backup/templates/configmap.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "openshift-etcd-backup.fullname" . }}
+  labels:
+    {{- include "openshift-etcd-backup.labels" . | nindent 4 }}
+data:
+  OCP_BACKUP_SUBDIR: {{ .Values.backup.subdir | quote }}
+  OCP_BACKUP_DIRNAME: {{ .Values.backup.dirname | quote }}
+  OCP_BACKUP_EXPIRE_TYPE: {{ .Values.backup.expiretype | quote }}
+  OCP_BACKUP_KEEP_DAYS: {{ .Values.backup.keepdays | quote }}
+  OCP_BACKUP_KEEP_COUNT: {{ .Values.backup.keepcount | quote }}
+  backup.script: |
+    #!/bin/sh
```

Comment:
please don't include a copy paste of the original upstream source for this script as a configmap